package.json

```diff
@@ -31,6 +31,10 @@
     "test": "mocha ./test/**/*.test.js",
```

Comment:
If we can get existing tests (those we wish to keep) working with Jest/Playwright, I think it would be fine to update the `"test"` script move the tests over in this PR, if you want to.

I'd like to see how the whole picture works, otherwise if we merge something now we're making a partially-informed decision (even if we're leaning that way).